app/views/account/show.html.erb

```diff
@@ -1,75 +1,72 @@
 <% content_for :title, t("account.your_account.heading") %>
 <% content_for :location, "your-account" %>
+<% content_for :account_navigation do %>
+<div class="govuk-grid-column-one-third">
```

Comment:
All of the `account_navigation` blocks seem to contain the same `govuk-grid-column-one-third`. Could this be moved to the parent so as to not be repeated throughout lots of templates?

Reply:
Good point! Have done this 👍